build/media-suite_compile.sh

```diff
@@ -1894,11 +1893,12 @@ if [[ $mpv != "n" ]] && pc_exists libavcodec libavformat libswscale libavfilter;
         add_third_party "https://github.com/KhronosGroup/SPIRV-Cross.git" spirv-cross
 
         # fix python indentation errors from non-existant code review
-        grep -ZRlP --include="*.py" '\t' third_party/spirv-tools/ | xargs -r -0 -n1 sed -i 's;\t;    ;g'
+        # grep -ZRlP --include="*.py" '\t' third_party/spirv-tools/ | xargs -r -0 -n1 sed -i 's;\t;    ;g'
```

Comment:
Why? This is to prevent issues from the committing broken python files as it has happened before.